src/Store/Features.re

```diff
@@ -662,18 +657,32 @@ let update =
     ({...state, changelog: model}, eff);
 
   // TODO: This should live in the editor feature project
-  | EditorFont(Service_Font.FontLoaded(font)) => (
+  | EditorFont(Service_Font.FontLoaded(font)) =>
+    let buffers' =
+      state.buffers |> IntMap.map(Oni_Core.Buffer.setFont(font));
+    (
       {
         ...state,
         editorFont: font,
+        buffers: buffers',
         layout:
           Feature_Layout.map(
-            editor => Feature_Editor.Editor.setFont(~font, editor),
+            editor => {
+              let bufferId = Feature_Editor.Editor.getBufferId(editor);
+              let updatedBuffer =
+                buffers'
+                |> Buffers.getBufferExn(bufferId)
+                |> Feature_Editor.EditorBuffer.ofBuffer;
+              Feature_Editor.Editor.updateBuffer(
+                ~buffer=updatedBuffer,
+                editor,
+              );
+            },
```

Comment:
As we discussed - we don't want to throw here. It would mean a crash for the user.

Reply:
Fixed in e8391a8!